Tests/Unit/Domain/Site/SiteRepositoryTest.php

```diff
@@ -89,7 +89,6 @@ public function canGetFirstAvailableSite(): void
         $this->assertThatSitesAreCreatedWithPageIds([333], [
             0 => ['language' => 0],
         ]);
-        $this->assertCacheIsWritten();
```

Comment:
Why removing that assertion?

Reply:
Caching of available site only happens while calling `getAvailableSites`. But as `hasAvailableSites` works with the generator directly "typo3ManagedSolrSites", no caching happens. That's why I have removed that check.